RELEASE.rst

```diff
@@ -68,13 +98,13 @@ Bug Fixes
 The following bug fixes were introduced:
 
 - A bug fix from C++ ``flagser`` on ``vertex_degree`` filtration has been propagated to pyflagser.
- 
+
```

Comment:
(Not important.) Out of curiosity, why this new line? It didn't seem necessary before.

Reply:
I feel like my editor automatically removed a line containing an empty space and added an empty line instead.

Reply:
I see :)